build.rs

```diff
@@ -10,6 +10,17 @@ fn main() {
     if rustc_minor_version().expect("Failed to get rustc version") >= 30 {
         println!("cargo:rustc-cfg=core_cvoid");
     }
+
+    let target = env::var("TARGET").unwrap();
+    let linkage = env::var("CARGO_CFG_TARGET_FEATURE").unwrap();
+
+    if target.contains("msvc") {
```

Comment:
As one final comment, I believe we'll want to disable this block for builds of libstd itself as the linkage is handled in the library itself, could feature detection be used to gate that?

Additionally, could a comment be placed here indicating what's happenign?

Reply:
Ah nowadays it's the `rustc-dep-of-std` feature, and only when that feature is not activated do we know we're building built from crates.io and we should include this snippet

Reply:
Alright, I've added a check for that feature and a comment.